samples/cpp/greedy_causal_lm/greedy_causal_lm.cpp

```diff
@@ -12,7 +12,8 @@ int main(int argc, char* argv[]) try {
     std::string device = "CPU";  // GPU can be used as well
 
     ov::genai::LLMPipeline pipe(models_path, device);
-    ov::genai::GenerationConfig config;
+    ov::genai::GenerationConfig config = pipe.get_generation_config();
+    config.do_sample = false;
```

Comment:
what's about other parameters like repetition penalty ?

Reply:
The sample preserves other parameters now. That's the reason for the PR.